kimanetwork/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "kima",
```

Comment:
The chain_name property must match the directory name. Either use kima and mimatestnet for both the property and directory names, or kimanetwork and kimanetworktestnet would work, too

Reply:
Thanks, assuming the `pretty_name` will take precedence with regards to frontends and UIs?

Reply:
Yes, that's right. this chain_name property should never be seen by users